libs/shared/sentry/src/lib/reporting.ts

```diff
@@ -56,6 +56,29 @@ export function isApolloError(err: Error): boolean {
   return false;
 }
 
+/**
+ * Determine if an error originates from auth-server. Auth server responds with error
+ * codes and numbers, and client applications handle these states accordingly. These
```

Comment:
You've dug in a lot more than I have here, but a couple of questions:
* There aren't _any_ auth-server errors we want logged to Sentry? I'm looking at 125, `'The request was blocked for security reasons'` which was helpful in seeing that problem with GQL/React or possibly the `'System unavailable, try again soon'` messages but maybe we can check Grafana for spikes/problems?
* Is this going to catch and not report any other errors that are not auth-server? I think profile-server errors follow this format for example, but I think they also go through auth-server, so probably doesn't count.

Reply:
- If we want to capture these errors, we should capture them in auth-server, not gql... Personally I don't think we need to report these since they are actually intentional error states.
- I don't think this is an issue. The general notion that errors with code & errno indicate error states that are used for client side flow control seems pretty solid to me.  If `auth-server` or `profile-server` want to capture or emit metrics for these types of errors, that seems fine, but I don't think GQL should be doing this. 

Another argument for why we shouldn't emit these from the `graphql-api` is that prior to introducing GQL these sorts of errors wouldn't have been emitted, so I don't I don't see why GQL should be emitting them now. It's true that some of these errors were useful to debug a previous situation, but the noise it is generating really isn't worth it, and we also had similar artifacts in our traces.

Reply:
I hadn't realized this was specifically for gql-api and not auth-server, but yeah agreed _if_ we wanted to capture these they should be in auth-server and not gql. Just wanted to do a sanity check - SGTM.